van-gogh-second-page/dalite/style.css

```diff
@@ -0,0 +1,84 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-weight: 500;
+}
+body{
+    background-color: #F497AF;
+    font-family: Gotham Rounded SSm,Arial,sans-serif;
```

Comment:
I think Gotham Rounded SSm has not imported from anywhere) You should import from google fonts or from a file downloaded from google fonts